app/src/main/java/com/example/taskyy/data/repository/LoginRepository.kt

```diff
@@ -0,0 +1,4 @@
+package com.example.taskyy.data.repository
+
+class LoginRepository {
+}
```

Comment:
I'd bundle multiple related calls in one repository, otherwise you'll end up with so many repositories quickly losing the overview. Something like `AuthRepository` for login, logout and authenticate makes sense :)